react/components/organisms/chat/chat.js

```diff
@@ -227,13 +228,18 @@ const styles = StyleSheet.create({
     },
     chatMessagesContainer: {
         flex: 1,
-        backgroundColor: "#f6f7f9"
+        backgroundColor: "#ffffff"
     },
     chatMessagesContent: {
-        paddingVertical: 12
+        paddingTop: 12,
+        marginBottom: -1

```

Comment:
is there an alernative to this?

Reply:
Nope, this is just so that the rich text input border does not appear as well as the last chat message border